src/execution/__tests__/subscribe-test.ts

```diff
@@ -135,9 +137,6 @@ async function expectPromise(promise: Promise<unknown>) {
   }
 
   return {
-    toReject() {
-      expect(caughtError).to.be.an.instanceOf(Error);
-    },
```

Comment:
Yes, it's a very bad practice to match random errors, because it results in a lot of false positive tests.